packages/sol6/src/yang/etsi-nfv-vnf.yang

```diff
@@ -890,7 +890,7 @@ submodule etsi-nfv-vnf {
         leaf size-of-storage {
           type uint64;
           units "Gb";
-          mandatory true;
+          default 0;
```

Comment:
Sorry. Just noticed this. Why is the default value 0, and not say 1. Are we expecting that NSD can do without any storage?

Reply:
The reason is that 0 is a magic value in OpenStack, meaning pick whatever size is in the image.

Reply:
And it works the same way (AFAIK) with VMware/OVA.

Reply:
Ahh! I see. In that case I am fine with all the changes. Thanks.